src/sentry/static/sentry/app/components/accordion.jsx

```diff
@@ -0,0 +1,81 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+import {Flex} from 'grid-emotion';
+import Badge from 'app/components/badge';
+import Button from 'app/components/buttons/button';
+
+class AccordionButton extends React.Component {
+  static propTypes = {
+    label: PropTypes.string.isRequired,
+    cutoff: PropTypes.number,
+    children: PropTypes.arrayOf(PropTypes.node).isRequired,
+  };
+
+  static defaultProps = {
+    cutoff: 0,
+  };
+
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      isOpen: false,
+    };
+  }
+
+  handleClick(e) {
+    e.preventDefault();
+    this.setState({isOpen: !this.state.isOpen});
+  }
+
+  render() {
+    const {label, cutoff, children} = this.props;
+
+    if (children.length <= cutoff + 1) return children;
```

Comment:
I think we'll want to use https://reactjs.org/docs/react-api.html#reactchildrencount so that we can use `Accordion` like:

```
<Accordion>
  <El>
  <El>
  <El>
</Accordion>
```

Will probably also need https://reactjs.org/docs/react-api.html#reactchildrentoarray

Reply:
dang good tips